tests/fixtures/ofxdata-schwab-brokerage.ofx

```diff
@@ -0,0 +1,461 @@
+OFXHEADER:100
```

Comment:
I noticed that this fixture has been added, but it is not actually run in the test suite - please could you add it? :) thanks!